xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+  </para>
+  
+  <screen>
+{
+  "create_default_networks": false,
+  //set this to your own internal domain
+  "dhcp_domain": "<replaceable>cloud.example.cisco-aci.lab</replaceable>",  
+  "use_lbaas": false,
+  "ml2_mechanism_drivers": [
+    "cisco_apic_ml2"
+  ],
+  "ml2_type_drivers": [
+    //this must be the first option
+    "opflex",  
+    //the second option defines the encapsulation used
+    "vxlan",  
+    "vlan"
+  ],
+  "ml2_type_drivers_default_provider_network": "opflex",
+  "ml2_type_drivers_default_tenant_network": "opflex",
+  "apic": {
+    //set this to your APIC IP address(es)
+    "hosts": "<replaceable>10.105.1.10,10.105.1.11</replaceable>",  
+    //set this to represent SOC as the APIC VMM domain
+    "system_id": "soc6",     
+    //use your own APIC login and password
+    "username": "<replaceable>admin</replaceable>", 
+    "password": "<replaceable>password</replaceable>"
+  },
+  "opflex": {
+     //use your own peer IP address and port
+    "peer_ip": "<replaceable>10.0.0.30</replaceable>", 
+    "peer_port": <replaceable>8009</replaceable>,
+    "encap": "vxlan",
+    "vxlan": {
+      "encap_iface": "br-int_vxlan0",
+      //get this value from your VLAN ID
+      "uplink_iface": "<replaceable>vlan.4093</replaceable>",  
+      "uplink_vlan": 4093,
+      //use your own remote IP address and port
+      "remote_ip": "<replaceable>10.0.0.32</replaceable>", 
+      "remote_port": <replaceable>8472</replaceable>
+    },
+  "apic_switches": {
+    "101": {
+      "switch_ports": {
+         //use your own hostname and switch port
+        "<replaceable>d58-00-00-00-00-00</replaceable>": { 
+          "switch_port": "<replaceable>1/2</replaceable>"
+        }
+      }
+    },
+    "102": {
+      "switch_ports": {
+        //use your own hostname and switch port
+        "<replaceable>d58-00-00-00-00-02</replaceable>": { 
+          "switch_port": "<replaceable>1/3</replaceable>"
+        }
+       }
+     }
+   }
+ }
+}
+  </screen>
+  
+  <para>
+      Save your changes and click Apply, and then apply the Neutron and Nova barclamps. If the Nova barclamp fails see <xref linkend="sec.troubleshooting"/>.
+  </para>
+  
+  <para>
+      Now you can verify that the VLAN is correctly configured on the compute nodes, and the network interface has an IP address from the ACI fabric.
+  </para>
+  <screen><prompt>root # </prompt><command>ip a show dev vlan.4093</command>
+</screen> 
+<para>
+    If the interface does not get an IP address from the ACI fabric, see the troubleshooting section.
+</para>
```

Comment:
Is it possible to provide a link to the troubleshooting sections that deals with this problem?